apollo-biz/src/test/java/com/ctrip/framework/apollo/biz/service/NamespaceBranchServiceTest.java

```diff
@@ -64,25 +64,32 @@ public void testUpdateBranchGrayRulesWithUpdateOnce() {
 
     namespaceBranchService.updateBranchGrayRules(testApp, testCluster, testNamespace, testBranchName, rule);
 
-    GrayReleaseRule
-        activeRule =
-        namespaceBranchService.findBranchGrayRules(testApp, testCluster, testNamespace, testBranchName);
+    GrayReleaseRule activeRule = namespaceBranchService.findBranchGrayRules(testApp, testCluster, testNamespace,
```

Comment:
It's better not to involve unnecessary changes, as they would make it hard to distinguish what's actually changed.

Reply:
Thanks for pointing out! It was caused by autoformatting and now I have restored them to the original version.